frontend/src/components/object/ObjectMetadata.vue

```diff
@@ -97,7 +97,10 @@ watch([props, tsGetMetadata,vsGetMetadata] , () => {
 </script>
 
 <template>
-  <div class="grid details-grid grid-nogutter mb-2">
+  <div
+    v-if="objectMetadata?.metadata.length"
```

Comment:
Double check with Tyler. I believe I didn't v-if this intentionally. Something about wanting to show the object had no metadata.